providers/discord/src/airflow/providers/discord/notifications/discord.py

```diff
@@ -39,18 +43,21 @@ class DiscordNotifier(BaseNotifier):
     :param username: The username to send the message as. Optional
     :param avatar_url: The URL of the avatar to use for the message. Optional
     :param tts: Text to speech.
+    :param embed: Discord embed object. See:
+           https://discord.com/developers/docs/resources/message#embed-object-embed-author-structure
     """
 
     # A property that specifies the attributes that can be templated.
-    template_fields = ("discord_conn_id", "text", "username", "avatar_url", "tts")
+    template_fields = ("discord_conn_id", "text", "username", "avatar_url", "tts", "embed")
 
     def __init__(
         self,
         discord_conn_id: str = "discord_webhook_default",
-        text: str = "This is a default message",
```

Comment:
Is it by API required to have a mandatory text? Can we rather add a validationor make this property mandatory instead of adding a default? (Or is the change committed by accident?)

Reply:
I think at least one of the following needs to be provided:
- content
- embeds
- sticker_ids
- components
- files[n]
- poll

However content can be an empty string.

See Discord API [doc](https://discord.com/developers/docs/resources/message#create-message-jsonform-params)